.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: python
-python: 
+python:
   - "2.7"
 
 install:
   - make bootstrap
 
 script:
-  - make test
-
+  - make test lint
```

Comment:
This could be another PR, but I needed that to ensure the CI runs the `lint` step.

Reply:
Actually, you could issue a PR right now with this small change and probably get it merged rather soon ;)